### ISSUE TITLE
Homework Assignments

### ISSUE THREAD

**Issue by aarmn** [cheraaqee/presenting_concepts#1]
Hello there,
Where can we log of sessions assignments?

**Comment by cheraaqee** [comment]
Hey,
They are usually listed in the session contents' ```.pdf``` files. (Is that what you wanted? :thinking: )

**Comment by aarmn** [comment]
> usually listed in the session contents

Right, but I'm worried that I'm not handing **all** of them, because as you mentioned, our class has some similarities to superposition state in quantum mechanics and my note-taking abilities are not among the best, therefore I rather ensure I have all the assignments from a source aside from verbal communication in class (if available) 
I would be thankful If you, prof, or any fellow classmate, can confirm all HWs are written in PDFs (aside from personal presentations)

**Comment by cheraaqee** [comment]
They've all been mentioned in the ```.pdf``` files so far. They probably will continue to be so ;-)
